zcash_client_backend/src/fees/standard.rs

```diff
@@ -0,0 +1,101 @@
+//! Change strategies designed for use with a standard fee.
```

Comment:
Why are we adding new strategies that are called "standard"? The standard is ZIP 317.

I see that we might want names that don't assume that we won't change the applicable ZIP again. However, for the time being I think this just obscures the fact that we need everyone to move to ZIP 317 ASAP. (We needed them to do that months ago. I'm grumpy about it, as you might be able to tell.)

Reply:
The main reason is that we need an enumerated type that defines the existing strategies, so we can correctly deserialize them when passed across the FFI boundary. Fully implementing ZIP 317 support requires UI changes that go beyond the scope of the change to the SDK, so we want to make it possible for wallet partners to make those UI changes on their own schedule.